go/v1beta1/storage/elasticsearch.go

```diff
@@ -488,28 +564,83 @@ func (es *ElasticsearchStorage) DeleteOccurrence(ctx context.Context, pID, oID s
 
 // GetNote returns the note with project (pID) and note ID (nID)
 func (es *ElasticsearchStorage) GetNote(ctx context.Context, pID, nID string) (*pb.Note, error) {
-	return nil, nil
+	log := es.logger.Named("GetNote")
```

Comment:
Asking @mrparkers to check this to make sure it matches his getOccurrence (i think that's what he was working on)

Reply:
It's similar enough.  I'll update some things after this is merged to be more consistent with the new `GetOccurrence` implementation.